cli-wrap/src/main/resources/cli/da-cli.py

```diff
@@ -339,6 +339,15 @@ def list_black(self):
         else:
             print("Bad arguments! For help use: ./da-cli.py --help")
             
+    def list_white(self):
+        if len(sys.argv) == 3:
+            self.da.print_white_artifacts()
+        elif len(sys.argv) == 4:
+             if self.pm.matchProd(sys.argv[3]):
+                self.da.print_white_artifacts(sys.argv[3])
+        else:
+            print("Bad arguments! For help use: ./da-cli.py --help")
+            
```

Comment:
I ran tests, and noticed that method for ./da-cli.py list white [PRODUCT_NAME:VERSION] is missing. I have no idea how i could overlook this so many times *shame*